src/aclk/README.md

```diff
@@ -12,24 +11,327 @@ For ACLK to function properly, your Agents need outbound access to Netdata Cloud
 |---------------------|----------------------------------------------------------------------------|
 | Domain              | • `app.netdata.cloud`<br/>• `api.netdata.cloud`<br/>• `mqtt.netdata.cloud` |
 
-> **Important**
->
-> IP addresses can change without notice! Always **prefer domain allowlisting**. If you must use IP addresses, be aware that they vary based on your geographic location due to CDN-edge servers. You'll need to regularly verify the IP addresses specific to your region.
+:::warning
+
+IP addresses can change without notice! Always **prefer domain allowlisting**. If you must use IP addresses, be aware that they vary based on your geographic location due to CDN-edge servers. You'll need to regularly verify the IP addresses specific to your region.
+
+:::
 
 ## Data privacy
 
 Your monitoring data belongs to you. Here's how we ensure this:
-
 - **Zero Metric Storage**: We do not store any metrics or logs in Netdata Cloud.
 - **Local Data Control**: All your monitoring data stays within your infrastructure.
 - **Minimal Metadata**: We store only essential metadata needed for coordination and access control.
 
 For complete transparency:
-
 - Read our detailed [Privacy Policy](https://netdata.cloud/privacy/)
 - Learn more about [stored metadata](/docs/netdata-cloud/README.md#stored-metadata)
 
 ## Enable and configure the ACLK
 
 The Agent-Cloud Link is enabled automatically—no configuration needed.
+
 If your Agent requires a proxy to access the internet, you'll need to [configure proxy settings](/src/claim/README.md#automatically-via-a-provisioning-system-or-the-command-line).
+
+## Troubleshooting
+
+### Check ACLK connection status
+
+Use these commands to verify your ACLK connection:
+
+```bash
+# Command-line status check
+sudo netdatacli aclk-state
+
+# JSON format for programmatic access
+sudo netdatacli aclk-state json
+
+# Web API endpoint
+curl http://localhost:19999/api/v1/aclk
+curl http://localhost:19999/api/v3/info
+```
+
+**Expected output:**
+```
+ACLK Available: Yes/No
+Claimed: Yes/No
+Claimed Id: <UUID>
+Online: Yes/No
+Cloud URL: https://app.netdata.cloud
+```
```

Comment:
It will also include 

```
ACLK Proxy: none
Publish Latency: 140ms 500us
```

if a proxy is defined and the most recent publish message latency. The detailed publish latency chart is available under Netdata / Agent / Nework traffic

netdata.aclk_puback_latency